doc/articles/howto-consume-webservices.md

```diff
@@ -1414,7 +1414,7 @@ You will start by adding the data models.
     public async Task<IEnumerable<Favorite>> GetAll()
     {
         var result = await this.GetAsync(
-            $"https://api.thecatapi.com/v1/favourites",
+            $"https://api.thecatapi.com/v1/favorites",
```

Comment:
This should not be changed, as well as the other references below

Reply:
Right, makes sense 👍 thanks! I'll fix this prior to review.